src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicMetricExporter.java

```diff
@@ -0,0 +1,186 @@
+package com.newrelic.telemetry.opentelemetry.export;
+
+import com.newrelic.telemetry.Attributes;
+import com.newrelic.telemetry.SimpleMetricBatchSender;
+import com.newrelic.telemetry.TelemetryClient;
+import com.newrelic.telemetry.metrics.Metric;
+import com.newrelic.telemetry.metrics.MetricBatchSenderBuilder;
+import com.newrelic.telemetry.metrics.MetricBuffer;
+import io.opentelemetry.sdk.internal.MillisClock;
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import io.opentelemetry.sdk.metrics.data.MetricData.Descriptor;
+import io.opentelemetry.sdk.metrics.data.MetricData.Descriptor.Type;
+import io.opentelemetry.sdk.metrics.data.MetricData.Point;
+import io.opentelemetry.sdk.metrics.export.MetricExporter;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.util.Collection;
+
+public class NewRelicMetricExporter implements MetricExporter {
```

Comment:
This seems like it might be important enough to warrant some high-level descriptive javadocs.

Reply:
yeah. this whole class is missing javadoc. ok to move this chore to an issue to be done before official release?